fractal-puppy/Engine
# The Graveyard bridges don't overlap the floor anymore

**Test**
- Open The Graveyard level
- Check if the bridges are navigable
- Check if the bridges are not overlapping the floor or any other asset
- Check that the level is fully playable
